src/components/playlistModal/PlaylistModal.tsx

```diff
@@ -38,16 +38,20 @@ export const PlaylistModal = ({
   const handleDescriptionChange = (description: string) =>
     setPlaylistForm({ ...playlistForm, description });
 
-  const validate = () => {
+  const validate = (): boolean => {
     const userTitle = playlistForm.title;
     if (userTitle == null || userTitle === '') {
       setTitleError(true);
+      return false;
     }
+    return true;
   };
 
   const onConfirm = () => {
-    validate();
-    handleConfirm(playlistForm.title, playlistForm.description);
+    const isValid = validate();
+    if (isValid) {
```

Comment:
originally wanted to use `titleError` but it is not yet set when we reach this point